packages/experimental/RadioGroup/src/__tests__/RadioGroupExperimental.test.tsx

```diff
@@ -1,11 +1,32 @@
 import * as React from 'react';
 import { RadioGroup } from '../RadioGroup/RadioGroup';
+import { Radio } from '../Radio/Radio';
+import { View } from 'react-native';
 import * as renderer from 'react-test-renderer';
 import { checkRenderConsistency, checkReRender } from '@fluentui-react-native/test-tools';
 
 describe('RadioGroup component tests', () => {
   it('RadioGroup default', () => {
-    const tree = renderer.create(<RadioGroup>Your component</RadioGroup>).toJSON();
+    const tree = renderer
+      .create(
+        <RadioGroup label="Uncontrolled RadioGroup" defaultValue="key2">
+          <Radio value="key1" label="RadioButton1" />
+          <Radio value="key2" label="RadioButton2" />
+        </RadioGroup>,
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('Radio not direct child of radio group', () => {
```

Comment:
Should there be tests for additional props like disabled?

Reply:
Added!